controllers/nnf_workflow_controller_helpers.go

```diff
@@ -691,9 +691,11 @@ func (r *NnfWorkflowReconciler) unmountNnfAccessIfNecessary(ctx context.Context,
 		},
 	}
 
+	// Check if the NnfAccess exists. If it doesn't, then we don't need to unmount anything. If we didn't
```

Comment:
It's kinda too bad we're going to be doing this for all workflows, not just data movements. I wonder if it's a performance concern.

Reply:
> It's kinda too bad we're going to be doing this for all workflows, not just data movements. I wonder if it's a performance concern.

It's just doing a Get() on the client cache, so I don't think there's a performance issue.